cmd/kubeadm/app/phases/apiconfig/clusterroles.go

```diff
@@ -64,7 +65,9 @@ func CreateServiceAccounts(clientset *clientset.Clientset) error {
 
 	for _, sa := range serviceAccounts {
 		if _, err := clientset.CoreV1().ServiceAccounts(metav1.NamespaceSystem).Create(&sa); err != nil {
-			return err
+			if !apierrors.IsAlreadyExists(err) {
+				return err
+			}
```

Comment:
Of course!

Yeah, that's correct. I did it mechanically locally without thinking of this special case... this SGTM